.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+
+python:
+  - "3.6"
+
+# The tests run only if it is an PR or if a tag is pushed. This should prevent duplicate
+# builds with pr and push. We cannot disable push as it also disables deployment via
+# tags.
+if: type != push OR tag IS present OR branch = master
+
+install:
+  - sudo apt-get update
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - PATH="$HOME/miniconda/bin:$PATH"
+  - conda env create -f configurations/environment.yml
+  - source activate structExpectations-analysis
+
+before_script:
+  - export PYTHONPATH=$PYTHONPATH:$(pwd)
```

Comment:
These two lines: This is a duplication from `.envrc`, you can simply source that file here as well. Now, I cannot simply run the same commands from the command line locally without further setup.